velox/exec/Exchange.h

```diff
@@ -121,9 +129,18 @@ class ExchangeQueue {
     auto page = std::move(queue_.front());
     queue_.pop_front();
     *atEnd = false;
+    byteSize_ -= page->byteSize();
     return page;
   }
 
+  uint64_t byteSize() const {
```

Comment:
nit: totalBytes might be a clearer name. Perhaps, add a small comment to clarify that this is the total number of bytes stored in the queue.